docs/operator.md

```diff
@@ -712,7 +712,7 @@ Specifies whether Service for HAProxy should [route internal traffic :octicons-l
 
 ### `proxy.haproxy.expose.loadBalancerIP`
 
-The static IP-address for the load balancer.
+The static IP-address for the load balancer. This field is deprecated starting with version 0.10.0. If you have defined it, refer to the [Kubernetes documentation :octicons-link-external-16:](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) for recommendations. 
 
```

Comment:
it is not depricated. I was removed.

Reply:
I'm not sure that we need word depricated here because it creates confusion. We can rewrite it : 
loadBalancerIP is deprecated in Kubernetes v1.24+ and removed starting with operator version 0.10.0

Reply:
Love the idea, fixed.